lib/portfinder.js

```diff
@@ -4,37 +4,21 @@
  * (C) 2011, Charlie Robbins
  *
  */
- 
+
 var fs = require('fs'),
     net = require('net'),
     path = require('path'),
     async = require('async'),
     mkdirp = require('mkdirp').mkdirp;
 
-//
-// ### @basePort {Number}
-// The lowest port to begin any port search from
-//
-exports.basePort = 8000;
+var internals = {};
 
-//
-// ### @basePath {string}
-// Default path to begin any socket search from
-//
-exports.basePath = '/tmp/portfinder'
-
-//
-// ### function getPort (options, callback)
-// #### @options {Object} Settings to use when finding the necessary port
-// #### @callback {function} Continuation to respond to when complete.
-// Responds with a unbound port on the current machine.
-//
-exports.getPort = function (options, callback) {
+internals.testPort = function(options, callback) {
```

Comment:
exporting it exposes the old api to the world that only checks the default port when calling net.createServer, which varies per system and leaves other interfaces//hosts on the same port open. 

The transplanted internals.testPort function is an exact copy of the original exports.getPort function so 2 other small goals(besides not exposing it) we're to: 1) Make code review easy 2) Allow the new getPort function easier to reason about.

I can inline it into the new getPort function if that's a better fit for the codebase. Not opposed.